app/utils/useAudio.js

```
@@ -0,0 +1,81 @@
+    audio.removeEventListener('timeupdate', handleTimeUpdate);
+    audio.removeEventListener('loadedmetadata', handleLoadedMetadata);
+    audio.removeEventListener('ended', handleEnded);
+  };
+}
+
+export function useAudio({ currentSong, isPlaying, volume, onSetCurrentTime, onSetDuration, onPause }) {
+  const audioRef = useRef(null);
+
+  // Update audio src
+  useEffect(() => {
+    updateAudioSource({audio: audioRef.current, song: currentSong, volume, isPlaying, onPause});
```

ESLint

[error] Replace `audio:·audioRef.current,·song:·currentSong,·volume,·isPlaying,·onPause` with `·audio:·audioRef.current,·song:·currentSong,·volume,·isPlaying,·onPause·`

(prettier/prettier)